addons/api/mirage/scenarios/ipc.js

```diff
@@ -147,9 +147,9 @@ export default function initializeMockIPC(server, config) {
    * main process, and routes messages to it in a way similar to our
    * preload.js script.
    *
-   * Initializes mock IPC only in a non-Electron non-testing context.
+   * Initializes mock IPC only in a non-testing context and when mirage is turned on.
    */
-  if (!config.isElectron && !isTesting) {
```

Comment:
Interesting, now I am thinking if this could fix a bug (or let's say bad behaviour). I will keep an eye on it. Thanks for it!